### PR TITLE
Disallows nicknames to be pattrbot

### LINKED ISSUE #9: Users can change their name to pattrbot
Obviously this is a big issue. 'Nuff said.

### PR DESCRIPTION
**Pull request by jeffw16**
Fixes #9

### CODE REVIEW

File: pattr.py [hunk 1]
```diff
@@ -106,6 +106,8 @@ def nick_passes(nickname):
         return False
     elif len(nickname) == 0:
         return False
+    elif nickname is 'pattrbot':
```
Comment (justinpotts) [inline pattr.py]: Would you mind making this
```
elif nickname == 'pattrbot':
```
to make it consistent with the rest of our statements?

**Comment by jeffw16** [comment]
Done, you can now merge.

**Comment by justinpotts** [comment]
Thanks for this @jeffw16! 
giphy

**Comment by jeffw16** [comment]
Lol no problem!

**Pull request merged**